Open training/src/Topic_0/House_director.java

```diff
@@ -0,0 +1,33 @@
+package Topic_0;
+
+public class House_director {
+
+	private House_concrete_builder house_builder = null;
+
+	public House_director(House_concrete_builder house) {
+		this.house_builder = house;
+	}
+
+	public void Construct_house() {
+		house_builder.buildFoundation();
```

Comment:
Usually builders methods allow you to use them like this:
```java
HouseBuilder someBuilder = new HouseBuilder();
House myHouse = someBuilder.buildRooms().buildFloor().buildWindows().buildServices().getHouse();
```
Can you modify your Builder to be able to use it like that?